charts/newrelic-k8s-metrics-adapter/templates/configmap.yaml

```diff
@@ -10,6 +10,7 @@ data:
     {{- with (include "newrelic-k8s-metrics-adapter.region" .) }}
     region: {{ . }}
     {{- end }}
+    cacheTTLSeconds: {{ .Values.config.cacheTTLSeconds | default "0" }}
```

Comment:
> As far as I know this default is used only if someone is passing config.cacheTTLSeconds=, that I would consider a blank value setting it to the chart default.
Also if they omit this parameter in a new `values.yaml` file.

> or 0 as in go since it is a null, I guess you put 0 for that)
Yes I didn't want to change de default behavior but to make it explicit in the chart.